erpc/http_server.go

```diff
@@ -37,20 +37,23 @@ type HttpServer struct {
 }
 
 func NewHttpServer(ctx context.Context, logger *zerolog.Logger, cfg *common.ServerConfig, admin *common.AdminConfig, erpc *ERPC) *HttpServer {
-	reqMaxTimeout, err := time.ParseDuration(*cfg.MaxTimeout)
-	if err != nil {
-		if cfg.MaxTimeout != nil && *cfg.MaxTimeout != "" {
-			logger.Error().Err(err).Msgf("failed to parse max timeout duration using 30s default")
+	err := error(nil)
```

Comment:
```suggestion
	err := nil
```
should work I suppose?